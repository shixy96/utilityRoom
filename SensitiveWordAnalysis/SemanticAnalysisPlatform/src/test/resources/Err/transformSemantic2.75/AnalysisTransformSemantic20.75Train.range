x
-1 1
1 0 12
2 0 14.75
3 0 24.75
4 0 16.899999999999999
5 0 18.5625
6 0 14.75
7 0 24
8 0 27.149999999999999
9 0 24
10 0 24.5
11 0 21.75
12 0 24.5
13 0 15.200000000000001
14 0 21.399999999999999
15 0 17.399999999999999
16 0 19
17 0 17.800000000000001
18 0 17.399999999999999
19 0 17.800000000000001
20 0 3
21 0 3
