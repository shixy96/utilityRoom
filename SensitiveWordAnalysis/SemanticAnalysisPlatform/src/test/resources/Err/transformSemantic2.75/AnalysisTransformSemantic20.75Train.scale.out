log2c=5 log2g=-7 rate=82.328
log2c=-1 log2g=-7 rate=74.5658
log2c=5 log2g=-1 rate=86.0131
log2c=-1 log2g=-1 rate=83.9091
log2c=11 log2g=-7 rate=82.9816
log2c=11 log2g=-1 rate=87.4883
log2c=5 log2g=-13 rate=74.5472
log2c=-1 log2g=-13 rate=74.5969
log2c=11 log2g=-13 rate=82.2222
log2c=-3 log2g=-7 rate=74.6032
log2c=-3 log2g=-1 rate=82.8634
log2c=-3 log2g=-13 rate=74.5969
log2c=5 log2g=1 rate=87.8431
log2c=-1 log2g=1 rate=86.6916
log2c=11 log2g=1 rate=87.6315
log2c=-3 log2g=1 rate=85.0545
log2c=9 log2g=-7 rate=82.6829
log2c=9 log2g=-1 rate=87.5257
log2c=9 log2g=-13 rate=82.2284
log2c=9 log2g=1 rate=87.8743
log2c=5 log2g=-11 rate=82.0666
log2c=-1 log2g=-11 rate=74.5969
log2c=11 log2g=-11 rate=82.2409
log2c=-3 log2g=-11 rate=74.5969
log2c=9 log2g=-11 rate=82.2409
log2c=3 log2g=-7 rate=82.2284
log2c=3 log2g=-1 rate=85.4093
log2c=3 log2g=-13 rate=74.6032
log2c=3 log2g=1 rate=88.0735
log2c=3 log2g=-11 rate=74.5534
log2c=5 log2g=-5 rate=82.7015
log2c=-1 log2g=-5 rate=82.1475
log2c=11 log2g=-5 rate=84.2266
log2c=-3 log2g=-5 rate=75.8917
log2c=9 log2g=-5 rate=83.8593
log2c=3 log2g=-5 rate=82.5148
log2c=15 log2g=-7 rate=83.9029
log2c=15 log2g=-1 rate=87.202
log2c=15 log2g=-13 rate=82.2035
log2c=15 log2g=1 rate=87.2207
log2c=15 log2g=-11 rate=82.521
log2c=15 log2g=-5 rate=85.0545
log2c=5 log2g=-15 rate=74.6032
log2c=-1 log2g=-15 rate=74.5969
log2c=11 log2g=-15 rate=82.2098
log2c=-3 log2g=-15 rate=74.5969
log2c=9 log2g=-15 rate=82.0355
log2c=3 log2g=-15 rate=74.5969
log2c=15 log2g=-15 rate=82.2347
log2c=-5 log2g=-7 rate=74.5969
log2c=-5 log2g=-1 rate=82.6891
log2c=-5 log2g=-13 rate=74.5969
log2c=-5 log2g=1 rate=83.1248
log2c=-5 log2g=-11 rate=74.5969
log2c=-5 log2g=-5 rate=74.5969
log2c=-5 log2g=-15 rate=74.5969
log2c=5 log2g=3 rate=88.4469
log2c=-1 log2g=3 rate=87.949
log2c=11 log2g=3 rate=88.0735
log2c=-3 log2g=3 rate=86.3492
log2c=9 log2g=3 rate=88.0984
log2c=3 log2g=3 rate=88.5652
log2c=15 log2g=3 rate=87.7498
log2c=-5 log2g=3 rate=84.9549
log2c=7 log2g=-7 rate=82.4712
log2c=7 log2g=-1 rate=86.897
log2c=7 log2g=-13 rate=82.0728
log2c=7 log2g=1 rate=87.7871
log2c=7 log2g=-11 rate=82.2284
log2c=7 log2g=-5 rate=83.1373
log2c=7 log2g=-15 rate=74.5534
log2c=7 log2g=3 rate=88.3287
log2c=5 log2g=-9 rate=82.2284
log2c=-1 log2g=-9 rate=74.6032
log2c=11 log2g=-9 rate=82.409
log2c=-3 log2g=-9 rate=74.5969
log2c=9 log2g=-9 rate=82.328
log2c=3 log2g=-9 rate=82.0728
log2c=15 log2g=-9 rate=82.5833
log2c=-5 log2g=-9 rate=74.5969
log2c=7 log2g=-9 rate=82.2409
log2c=1 log2g=-7 rate=82.1662
log2c=1 log2g=-1 rate=84.3386
log2c=1 log2g=-13 rate=74.5969
log2c=1 log2g=1 rate=87.4634
log2c=1 log2g=-11 rate=74.6032
log2c=1 log2g=-5 rate=82.3529
log2c=1 log2g=-15 rate=74.5969
log2c=1 log2g=3 rate=88.6399
log2c=1 log2g=-9 rate=74.5596
log2c=5 log2g=-3 rate=84.0959
log2c=-1 log2g=-3 rate=82.5335
log2c=11 log2g=-3 rate=85.3284
log2c=-3 log2g=-3 rate=82.3903
log2c=9 log2g=-3 rate=84.8366
log2c=3 log2g=-3 rate=83.5481
log2c=15 log2g=-3 rate=86.8908
log2c=-5 log2g=-3 rate=78.5434
log2c=7 log2g=-3 rate=84.7121
log2c=1 log2g=-3 rate=82.9505
log2c=13 log2g=-7 rate=83.6975
log2c=13 log2g=-1 rate=87.563
log2c=13 log2g=-13 rate=82.3343
log2c=13 log2g=1 rate=87.6128
log2c=13 log2g=-11 rate=82.409
log2c=13 log2g=-5 rate=84.625
log2c=13 log2g=-15 rate=82.1973
log2c=13 log2g=3 rate=87.9365
log2c=13 log2g=-9 rate=82.6393
log2c=13 log2g=-3 rate=86.094
